app/controllers/concerns/staff_response_context.rb

```diff
@@ -22,10 +22,18 @@ def nomis_checker
 
   def prisoner_details
     @prisoner_details ||= PrisonerDetailsPresenter.new(
-      prisoner_validation, prisoner_location
+      prisoner_validation
     )
   end
 
+  def nomis_info
+    @nomis_info ||= NomisInfo.new(prisoner_validation, prisoner_location_validation)
+  end
+
+  def prisoner_location
+    @prisoner_location ||= PrisonerLocation.new(prisoner_location_validation)
```

Comment:
Is PrisonerLocation a presenter (like PrisonerDetailsPresenter)

Reply:
Yup, just "presentered" up *all* of the presenters